testing/environments/snapshot.yml

```diff
@@ -26,7 +26,7 @@ services:
       - ./docker/logstash/pki:/etc/pki:ro
 
   kibana:
-    image: docker.elastic.co/kibana/kibana:7.1.0-SNAPSHOT
+    image: docker.elastic.co/kibana/kibana:7.0.0-SNAPSHOT
```

Comment:
This PR should not change which version of the stack gets pulled by the master branch.

With the 7.0 branch being cut, master should be on stack 7.1.

I'm also curious about your comment in commit 38f67c1. The PR you reference should be included in 7.0.0 beta, at this time. Perhaps you need a good cleanup of your cached Docker images?